otto.go

```diff
@@ -655,6 +655,25 @@ func (self Object) Keys() []string {
 	return keys
 }
 
+// Get the keys (and those of the parents) for the object, in order of
+// "closest" to "furthest"
+func (self Object) KeysByParent() [][]string {
```

Comment:
Would be nice if we started to pass linting with doc comments e.g. this should be of the form:
```go
// KeysByParent returns the keys for the object, and those if its parents, in order of "closest" to "furthest"
```